sprites.py

```diff
@@ -268,7 +269,7 @@ def set_layer(self, layer=None):
     def set_label(self, new_label, i=0):
         ''' Set the label drawn on the sprite '''
         self._extend_labels_array(i)
-        if type(new_label) is str or type(new_label) is unicode:
+        if type(new_label) is str or type(new_label) is str:
```

Comment:
You can delete this `type(new_label) is str` it is repeated twice.

Reply:
Thanks for noticing, done in a517b91 along with other changes.